package.json

```diff
@@ -17,5 +17,9 @@
   },
   "dependencies": {
     "rxjs": "^5.0.2"
+  },
+  "devDependencies": {
+    "react-native": "^0.43.2",
```

Comment:
IMHO we should exclude both as devDependency, they should be part of the project that uses this library and `npm link` should be sufficient for development.

No, we should not force projects to use `react-native-windows`. As far as I know the files just don't get included in the project.